LoopKit/DeviceManager/PumpManager.swift

```diff
@@ -169,6 +169,17 @@ public protocol PumpManager: DeviceManager {
     /// - Parameters:
     ///   - rate: The maximum rate the pumpmanager should expect to receive in an enactTempBasal command.
     func setMaximumTempBasalRate(_ rate: Double)
+
+    /// Sync the schedule of basal rates to the pump, annotating the result with the proper time zone.
+    ///
+    /// - Precondition:
+    ///   - `scheduleItems` must not be empty.
+    ///
+    /// - Parameters:
+    ///   - scheduleItems: The items comprising the basal rate schedule
+    ///   - completion: A closure called after the command is complete
+    ///   - result: A BasalRateSchedule or an error describing why the command failed
+    func syncBasalRateSchedule(items scheduleItems: [RepeatingScheduleValue<Double>], completion: @escaping (_ result: Result<BasalRateSchedule, Error>) -> Void)
```

Comment:
[nit] Does this assume that the `scheduleItems` will always be consumed in the pump's time zone? It's not clear to me whether that or there would be some translation from phone's current time zone to pump's. Might want to add a comment one way or the other. Or perhaps that is already a basic assumed when communicating with the pump. If so, then it's fine.

Reply:
Current pump managers do set the new basal schedule using the phone's current timezone offset (`TimeZone.currentFixed`), and then send a `PumpManagerStatus` update to the `PumpManagerDelegate`, which Loop then uses to record the timezone change. This is not strictly required by the protocol, but it is a standard side effect. I think it's fine to keep this method clear of TimeZone specification, as the PumpManager does expose the TimeZone, and reports any changes, and a schedule should always be interpreted with respect to the PumpManager's timezone.

Reply:
Sorry, it's *not* a standard side effect. Timezone is only changed during a manual `setTime` command. This doesn't change the interpretation of a schedule being interpreted by the Pump's timezone. I'm just clarifying that it looks like the PumpManagers currently do not update their timezone in response to a basal schedule sync.